app/validation/validators.py

```diff
@@ -0,0 +1,129 @@
+from datetime import datetime
+
+from app.validation.error_messages import error_messages
+
+from wtforms import validators
+
+
+class IntegerCheck(object):
```

Comment:
The only validator that WTForms supplies that we've created our own version of is the number range - but that's because we want to throw a different error for each limit. There is no Integer validator in wtforms. As we'd need to get right into the call in order to change the message for each limit, this is the simplest solution.